### ISSUE TITLE
I am getting this error on custom data "input channels does not match filter's input channels" the images are of size (300,300,3)

### ISSUE THREAD

**Issue by utkarshshukla2912** [JiahuiYu/generative_inpainting#206]
`Traceback (most recent call last):
  File "train.py", line 47, in <module>
    images, config=config)
  File "/Users/utkarsh_1/Documents/Machine_Learning/AI_Tech/channel_reconstruction/scripts/generative_inpainting/inpaint_model.py", line 153, in build_graph_with_losses
    padding=config.PADDING)
  File "/Users/utkarsh_1/Documents/Machine_Learning/AI_Tech/channel_reconstruction/scripts/generative_inpainting/inpaint_model.py", line 97, in build_inpaint_net
    x, offset_flow = contextual_attention(x, x, mask_s, 3, 1, rate=2)
  File "/Users/utkarsh_1/Documents/Machine_Learning/AI_Tech/channel_reconstruction/scripts/generative_inpainting/inpaint_ops.py", line 307, in contextual_attention
    yi = tf.nn.conv2d_transpose(yi, wi_center, tf.concat([[1], raw_fs[1:]], axis=0), strides=[1,rate,rate,1]) / 4.
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/tensorflow/python/ops/nn_ops.py", line 1228, in conv2d_transpose
    filter.get_shape()[3]))
ValueError: input channels does not match filter's input channels, 1369 != 1444`

**Comment by JiahuiYu** [comment]
Hey @utkarshshukla2912 , I am here to help. Could you provide more information about the background? Are you training or testing? Have you modified the code? Which tensorflow version you are using?

I don't think other users have such problem.

**Comment by pandaadb** [comment]
Hi,

I am having the exact same issue. I am using images of size 900x300 (w x h) and getting the error as well: 
 
ValueError: input channels does not match filter's input channels, 4144 != 4294

Note, i created a custom mask too, by doing: 

```
mask = cv2.imread('/home/artur/dev/repo/generative_inpainting/date_mask_bg_small.png', 0)[np.newaxis,:,:,np.newaxis]
mask = np.float32(mask) / float(255.)
```
This is just a custom free form mask for my purpose. The error reproduces with a bounding box mask as well. 

Info about what I am doing: 

1. I am training (or trying). I tried existing models and that worked fine
2. I did modify the code in order to replace the mask you created with a custom mask for my images. 
3. The background images are my own images. I am currently only doing a CPU test run for about 10 images. I process them by loading them and then cropping random windows of 900x300 out of them. The size is chosen because it is the mask i will be using for actual processing later. 
4. Version of tensorflow is 1.7.0. I did try with the latest version and ran into the same issue. 

I am still playing around but my experience with tensorflow is relatively limited so any pointer will be helpful :) 

Thanks! 

Artur
